From caa6a3c9271b01b11ca1e80bdbf87e500bb0438f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sat, 15 Oct 2016 00:28:42 +0200
Subject: [PATCH] menus: Don't exclude any desktop files

Otherwise these are listed in the "Other" section, which is not the proper place.
---
 data/menus/gnome-flashback-applications.menu | 96 +---------------------------
 1 file changed, 1 insertion(+), 95 deletions(-)

diff --git a/data/menus/gnome-flashback-applications.menu b/data/menus/gnome-flashback-applications.menu
index a267f4f..0372835 100644
--- a/data/menus/gnome-flashback-applications.menu
+++ b/data/menus/gnome-flashback-applications.menu
@@ -151,54 +151,7 @@
       <Include>
         <And>
           <Category>Settings</Category>
-          <Not>
-            <Or>
-              <Category>System</Category>
-              <Category>X-GNOME-Settings-Panel</Category>
-              <Category>X-Unity-Settings-Panel</Category>
-              <Filename>alacarte.desktop</Filename>
-              <Filename>caribou.desktop</Filename>
-              <Filename>dconf-editor.desktop</Filename>
-              <Filename>fedora-im-chooser.desktop</Filename>
-              <Filename>fedora-release-notes.desktop</Filename>
-              <Filename>firewall-config.desktop</Filename>
-              <Filename>flash-player-properties.desktop</Filename>
-              <Filename>gconf-editor.desktop</Filename>
-              <Filename>gnome-abrt.desktop</Filename>
-              <Filename>fedora-abrt.desktop</Filename>
-              <Filename>gnome-orca.desktop</Filename>
-              <Filename>gnome-power-statistics.desktop</Filename>
-              <Filename>gnome-user-share-properties.desktop</Filename>
-              <Filename>ibus.desktop</Filename>
-              <Filename>ibus-daemon.desktop</Filename>
-              <Filename>ibus-setup-anthy.desktop</Filename>
-              <Filename>ibus-setup.desktop</Filename>
-              <Filename>ibus-setup-hangul.desktop</Filename>
-              <Filename>ibus-setup-libbopomofo.desktop</Filename>
-              <Filename>ibus-setup-libpinyin.desktop</Filename>
-              <Filename>ibus-setup-m17n.desktop</Filename>
-              <Filename>ibus-setup-typing-booster.desktop</Filename>
-              <Filename>im-chooser.desktop</Filename>
-              <Filename>itweb-settings.desktop</Filename>
-              <Filename>jhbuild.desktop</Filename>
-              <Filename>javaws.desktop</Filename>
-              <Filename>java-1.7.0-openjdk-jconsole.desktop</Filename>
-              <Filename>java-1.7.0-openjdk-policytool.desktop</Filename>
-              <Filename>log4j-chainsaw.desktop</Filename>
-              <Filename>log4j-logfactor5.desktop</Filename>
-              <Filename>nm-connection-editor.desktop</Filename>
-              <Filename>orca.desktop</Filename>
-              <Filename>setroubleshoot.desktop</Filename>
-              <Filename>authconfig.desktop</Filename>
-              <Filename>system-config-date.desktop</Filename>
-              <Filename>system-config-firewall.desktop</Filename>
-              <Filename>system-config-keyboard.desktop</Filename>
-              <Filename>system-config-language.desktop</Filename>
-              <Filename>system-config-printer.desktop</Filename>
-              <Filename>system-config-users.desktop</Filename>
-              <Filename>vino-preferences.desktop</Filename>
-            </Or>
-          </Not>
+          <Not><Category>System</Category></Not>
         </And>
       </Include>
     </Menu>
@@ -209,53 +162,6 @@
         <And>
           <Category>Settings</Category>
           <Category>System</Category>
-          <Not>
-            <Or>
-              <Category>X-GNOME-Settings-Panel</Category>
-              <Category>X-Unity-Settings-Panel</Category>
-              <Filename>alacarte.desktop</Filename>
-              <Filename>caribou.desktop</Filename>
-              <Filename>dconf-editor.desktop</Filename>
-              <Filename>fedora-im-chooser.desktop</Filename>
-              <Filename>fedora-release-notes.desktop</Filename>
-              <Filename>firewall-config.desktop</Filename>
-              <Filename>flash-player-properties.desktop</Filename>
-              <Filename>gconf-editor.desktop</Filename>
-              <Filename>gnome-abrt.desktop</Filename>
-              <Filename>fedora-abrt.desktop</Filename>
-              <Filename>gnome-orca.desktop</Filename>
-              <Filename>gnome-power-statistics.desktop</Filename>
-              <Filename>gnome-user-share-properties.desktop</Filename>
-              <Filename>ibus.desktop</Filename>
-              <Filename>ibus-daemon.desktop</Filename>
-              <Filename>ibus-setup-anthy.desktop</Filename>
-              <Filename>ibus-setup.desktop</Filename>
-              <Filename>ibus-setup-hangul.desktop</Filename>
-              <Filename>ibus-setup-libbopomofo.desktop</Filename>
-              <Filename>ibus-setup-libpinyin.desktop</Filename>
-              <Filename>ibus-setup-m17n.desktop</Filename>
-              <Filename>ibus-setup-typing-booster.desktop</Filename>
-              <Filename>im-chooser.desktop</Filename>
-              <Filename>itweb-settings.desktop</Filename>
-              <Filename>jhbuild.desktop</Filename>
-              <Filename>javaws.desktop</Filename>
-              <Filename>java-1.7.0-openjdk-jconsole.desktop</Filename>
-              <Filename>java-1.7.0-openjdk-policytool.desktop</Filename>
-              <Filename>log4j-chainsaw.desktop</Filename>
-              <Filename>log4j-logfactor5.desktop</Filename>
-              <Filename>nm-connection-editor.desktop</Filename>
-              <Filename>orca.desktop</Filename>
-              <Filename>setroubleshoot.desktop</Filename>
-              <Filename>authconfig.desktop</Filename>
-              <Filename>system-config-date.desktop</Filename>
-              <Filename>system-config-firewall.desktop</Filename>
-              <Filename>system-config-keyboard.desktop</Filename>
-              <Filename>system-config-language.desktop</Filename>
-              <Filename>system-config-printer.desktop</Filename>
-              <Filename>system-config-users.desktop</Filename>
-              <Filename>vino-preferences.desktop</Filename>
-            </Or>
-          </Not>
         </And>
       </Include>
     </Menu>
-- 
2.10.0

